include/deal.II/base/vectorization.h

```diff
@@ -114,13 +123,24 @@ class VectorizedArrayIterator
     return this->lane != other.lane;
   }
 
+  /**
+   * Copy assignment.
+   */
+  VectorizedArrayIterator<T>
+  operator=(const VectorizedArrayIterator<T> &other)
```

Comment:
You might want to consider in a second step to make this just `=default` because that's what your implementation actually does.